.eslintrc.json

```diff
@@ -8,5 +8,7 @@
         "ecmaVersion": "latest",
         "sourceType": "module"
     },
-    "rules": {}
+    "rules": {
+        "no-console": "off"
```

Comment:
```suggestion
        "no-console": ["error", { allow: ["warn", "error"] }]
```

Reply:
if that's what you're looking for anyway. Certainly don't think we should be allowing `console.log`

Reply:
hmm it gets messy implementing this globally. Inspiration taken from [eslint docs](https://eslint.org/docs/latest/rules/no-console#when-not-to-use-it). The config would end up looking like 
```json
 "rules": {
        "no-console": "off",
        "no-restricted-syntax": [
            "warn",
            {
                "selector": "CallExpression[callee.object.name='console'][callee.property.name!=/^(warn|error)$/]",
                "message": "Unexpected console statement"
            }
        ]
    }
```

Reply:
that's only for if you want to allow console overwrites but not console calls no? 

my snippet does work for erroring on console.log but not console.error

Reply:
ok so your snippet is not valid json hence why I went down the docs route. This snippet works...
```json
 "no-console": [
            "error",
            {
                "allow": [
                    "warn",
                    "error"
                ]
            }
        ]
```

Reply:
you miss two quote marks and the world goes mad :(